oci/pull.bzl

```diff
@@ -220,7 +221,7 @@ oci_pull_rule = repository_rule(
     implementation = _oci_pull_impl,
     attrs = {
         "image": attr.string(doc = "The name of the image we are fetching, e.g. gcr.io/distroless/static", mandatory = True),
-        "digest": attr.string(doc = "The digest of the manifest file", mandatory = True),
+        "identifier": attr.string(doc = "The digest or tag of the manifest file", mandatory = True),
```

Comment:
nit: `specifier` might be a better name

Reply:
I got identifier from here
https://github.com/google/go-containerregistry/blob/62f183e54939eabb8e80ad3dbc787d7e68e68a43/pkg/v1/remote/descriptor.go#L234
is there another spec location that's better?